docs/docs/recipes/styling-css.md

```diff
@@ -415,3 +415,44 @@ _NOTE: So for the above example, the relevant CSS declaration would be `font-fam
 - [Typography.js](/docs/typography-js/) - Another option for using Google fonts on a Gatsby site
 - [The Typefaces Project Docs](https://github.com/KyleAMathews/typefaces/blob/master/README.md)
 - [Live example on Kyle Mathews' blog](https://www.bricolage.io/typefaces-easiest-way-to-self-host-fonts/)
+
+## Using FontAwesome
+
+Using [FontAwesome](https://fontawesome.com/) gives you access to thousands of icons for use on your site. Since Gatsby sites are React sites, we recommend you use the [react-fontawesome](https://github.com/FortAwesome/react-fontawesome) SVG library.
```

Comment:
```suggestion
Using [FontAwesome](https://fontawesome.com/) gives you access to thousands of icons for use on your site. Since Gatsby sites are React sites, it's recommended to use the [react-fontawesome](https://github.com/FortAwesome/react-fontawesome) SVG library.
```